packages/core/src/checkout-buttons/strategies/googlepay/googlepay-button-strategy.ts

```diff
@@ -96,7 +97,7 @@ export default class GooglePayButtonStrategy implements CheckoutButtonStrategy {
         const googlePayButton = this._googlePayPaymentProcessor.createButton(
             handleValidButtonClick,
             buttonType,
-            buttonColor,
+            buttonColor || color,
```

Comment:
Could we also update the related unit test(s) to cover this change?

Reply:
@bc-peng I think we can make it in other way. Let me check some idea with @andriiVitvitskyi1990 first

Reply:
We can make it without adding any changes in checkout sdk